### DISCUSSION TITLE
System page does not show version number anymore

### DISCUSSION THREAD

**Discussion by johndoe0815** [TandoorRecipes/recipes#2776] [General]
Since a few releases, the system page does not show anymore which version is currently running. This information used to be included in earlier versions, but then it disappeared and never got added anymore. Is there a specific reason? I find it quite helpful to see which version I am running.
![grafik](https://github.com/TandoorRecipes/recipes/assets/92877294/4e2722a0-664d-49e4-9cb2-09f01f5169f6)

**Comment by smilerz** [comment]
a fix has been submitted for this
